package.json

```diff
@@ -18,5 +18,8 @@
   },
   "engines": {
     "node": "^14.15"
+  },
+  "dependencies": {
+    "lodash": "^4.17.21"
```

Comment:
uh oh! try to never add dependencies to the _root_ package.json. If you want lodash, add it to `server` or `client` independently. the same holds true if you want lodash on the frontend and backend. These are because they are entirely different environments with their own tooling